app/ports/templates/ports/stats_faq.html

```diff
@@ -0,0 +1,76 @@
+{% extends 'ports/layout.html' %}
+
+{% block title %}FAQ - Statistics |{% endblock %}
+
+{% block content %}
+    <br>
+    <h3>Frequently Asked Questions - Statistics</h3>
+    <hr>
+    <br>
+    <a name="collected-data"></a>
+    <h4>Which Statistics Are Collected?</h4>
+    <hr>
+    <p>The MacPorts statistics collects and submits the following information to the server:</p>
+    <ul>
+        <li>
+            <strong>A unique identifier for your MacPorts installation</strong><br/>
+            This identifier is generated by <code>uuidgen</code> when you first
+            install the statistics service. We use this identifier to overwrite
+            statistics data you submitted earlier – we keep at most one set of
+            statistics per user per month. The identifier does not convey any
+            information about your system or location.
+        </li>
+        <li>
+            <strong>Version numbers and system properties</strong><br/>
+            The statistics service collects and submits the versions of MacPorts,
+            OS X, GCC and Xcode installed on your system. It also tells us what
+            your CPU architecture, platform and build target architecture are.
+        </li>
+        <li>
+            <strong>A list of your installed ports</strong><br/>
+            The statistics daemon collects and sends a list of the ports you have
+            installed containing the following information for each port:
+            <ul>
+                <li>name and installed version</li>
+                <li>whether the port is active or inactive</li>
+                <li>chosen variants</li>
+            </ul>
+        </li>
+    </ul>
```

Comment:
We could add a sentence saying something like "You can check what gets sent by running `/opt/local/libexec/bin/mpstats show`". (Please check the path, I'm not behind my mac right now.)

Reply:
Yes, I was thinking of somehow including both the commands: `show/ submit`. Thank you.